Formula/ghostz.rb

```diff
@@ -13,11 +13,25 @@ class Ghostz < Formula
     sha256 "ce5f132a6ed9a711bffae6648b18f701f55d965e611ef7320847aee24061a19e" => :x86_64_linux
   end
 
-  depends_on "gcc" # needs openmp
-
-  fails_with :clang # needs openmp
+  on_macos do
+    depends_on "libomp"
+  end
 
   def install
+    if OS.mac?
+      ENV.cxx11
```

Comment:
What's the error message without this line? I'm still curious what it actually does. I thought that `-std=c++11` was the default for most modern compilers.

Reply:
```
ext/seg/src/seg.cpp:424:18: error: use of overloaded operator '==' is ambiguous (with operand types 'std::shared_ptr<SeqmentType>' and 'long')
                if (temp->next == NULL) {
                    ~~~~~~~~~~ ^  ~~~~
```

I kinda just threw in `ENV.cxx11` to see if it would fix the error

Reply:
Change `NULL` to `nullptr`.

Reply:
> I kinda just threw in ENV.cxx11 to see if it would fix the error.

Did it fix the error?

Reply:
> > I kinda just threw in ENV.cxx11 to see if it would fix the error.
> 
> Did it fix the error?

Yeah

> Change `NULL` to `nullptr`.

That worked 🤔